README.md

```diff
@@ -1,109 +1,46 @@
 # cvs-tsk-update-test-stations
 
-The template for CVS lambda services
+AWS Lambda function that updates the list of Automated Test Facilities (ATFs / Test Stations) in the ATF Sites DynamoDB from the master list in Dynamics CE.
 
-## Dependencies
-
-The project runs on node 10.x with typescript and serverless framework. For further details about project dependencies, please refer to the `package.json` file.
-[nvm](https://github.com/nvm-sh/nvm/blob/master/README.md) is used to managed node versions and configuration explicitly done per project using an `.npmrc` file.
-
-## Running the project
-
-Once the dependencies are installed, you will be required to rename the `/config/env.example` file to `.env.local` as we use dotenv files for configuration for local local development for example. Further information about [variables](https://www.serverless.com/framework/docs/providers/aws/guide/variables/) and [environment variables](https://www.serverless.com/framework/docs/environment-variables/) with serverless.
-Please note that multiple `.env` files can be created per environments. Our current development environment is 'local'.
-
-The application runs on port `:3001` by default when no stage is provided.
+Designed to be invoked by a timer every night to pick up the previous days changes, but can be invoked manually.
 
-To get started, please change the meta data of your `package.json` and `serverless.yml` file accordingly as well as base api route for your express router.
-The router is currently being mounted in the following file `src/infrastructure/api/index.ts`.
+## Description
 
-The service has local environmental variables (please see `env` placeholder file) set locally however should we wish to further extend the service, the environmental variables will need to be ported over to the CI/CD pipeline which currently uses `BRANCH` and `BUCKET`.
+The function authenticates with Azure AD, and uses the returned token to retrieve the details of ATFs that have changed from Dynamics CE using OData. Each updated ATF is added to EventBridge as a separate event.
```

Comment:
Might want to change the use of ATF to test station as we're going to be dealing with different types of test stations (including ATFs)

Reply:
thanks @aaron-downing - I've replaced all references to 'ATF' with 'test station'